test/unit/test_materialization.py

```diff
@@ -238,71 +242,143 @@ def test_materialization_status_structure(conn):
 def test_materialization_status_values(conn):
     with conn() as cnx, cnx.cursor(DictCursor) as cur:
         rows = cur.execute(
-            "select user_schema, user_view, partition, range_start, range_end from admin.materialization_status"
+            "select user_schema, user_view, partition, last_full_start, last_full_end, last_full_status, range_start, range_end from admin.materialization_status"
         ).fetchall()
 
         for row in rows:
             assert row["USER_SCHEMA"] is not None
             assert row["USER_VIEW"] is not None
             if row["USER_VIEW"] == "WAREHOUSE_LOAD_HISTORY":
                 assert row["PARTITION"] is not None
-            for c in ["RANGE_START", "RANGE_END"]:
+
+            assert row["LAST_FULL_STATUS"] == "SUCCESS"
+            for c in ["LAST_FULL_START", "LAST_FULL_END"]:
```

Comment:
Updated these checks to verify this as we have downstream checks that ensure we have a full materialization